examples/siro_sandbox/sandbox_app.py

```diff
@@ -140,59 +128,64 @@ def __init__(self, args, config, gui_input):
             self.gym_habitat_env, config, args, gui_input, self._step_recorder
         )
 
-        self.gui_agent_ctrl = self.ctrl_helper.get_gui_agent_controller()
-
-        self.cam_transform = None
-        self.cam_zoom_dist = 1.0
```

Comment:
These are examples of member variables related to our mouse-controlled camera functionality. All of this is now encapsulated in `camera_helper.py`.